docker/postgres-tools-9.6/Dockerfile

```diff
@@ -1,15 +1,15 @@
-FROM postgres:9.6-alpine
+FROM postgres:bullseye
```

Comment:
Ha! I was supposed to delete this image 8 months ago, right around the time I switched laptops I think 🤔 Might have a branch lying somewhere - will follow up 👍🏼